pkg/plugins/policies/meshloadbalancingstrategy/api/v1alpha1/meshloadbalancingstrategy.go

```diff
@@ -38,8 +39,75 @@ type LocalityAwareness struct {
 	// Disabled allows to disable locality-aware load balancing.
 	// When disabled requests are distributed across all endpoints regardless of locality.
 	Disabled *bool `json:"disabled,omitempty"`
+	// LocalZone defines locality aware load balancing priorities between dataplane proxies inside a zone
+	LocalZone *LocalZone `json:"localZone,omitempty"`
+	// CrossZone defines locality aware load balancing priorities when dataplane proxies inside local zone
+	// are unavailable
+	CrossZone *CrossZone `json:"crossZone,omitempty"`
+	// FailoverThreshold defines the percentage of live destination dataplane proxies below which load balancing to the
+	// next priority starts.
+	// Example: If you configure failoverThreshold to 70, and you have deployed 10 destination dataplane proxies.
+	// Load balancing to next priority will start when number of live destination dataplane proxies drops below 7.
+	// Default 70
+	FailoverThreshold *FailoverThreshold `json:"failoverThreshold,omitempty"`
 }
 
+type LocalZone struct {
+	// AffinityTags list of tags for local zone load balancing.
+	AffinityTags []AffinityTag `json:"affinityTags"`
+}
+
+type AffinityTag struct {
+	// Key defines tag for which affinity is configured
+	Key string `json:"key"`
+	// Weight of the tag used for load balancing. The bigger the weight the bigger the priority.
+	// Percentage of local traffic load balanced to tag is computed by dividing weight by sum of weights from all tags.
+	// For example with two affinity tags first with weight 80 and second with weight 20,
+	// then 80% of traffic will be redirected to the first tag, and 20% of traffic will be redirected to second one.
+	// Setting weights is not mandatory. When weights are not set control plane will compute default weight based on list order.
```

Comment:
We can add info about defaults 90%, 9% .....

Reply:
added

Reply:
Maybe we can say:

```
// Default: If you don't specify weight we will set that 90% of the traffic goes to the first entry, 9% to the next, and lower for another.
```

We could even give a math rule: `max(1, 9* 10^(len of rules - rule index))`